task/07-yield-tasks.js

```diff
@@ -33,7 +33,27 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+    for(let i = 99; i >= 1; i--)
+    {
+        if(i == 1)
+        {
+            yield `${i} bottle of beer on the wall, ${i} bottle of beer.`
+            yield `Take one down and pass it around, no more bottles of beer on the wall.`
+            yield 'No more bottles of beer on the wall, no more bottles of beer.';
+            yield 'Go to the store and buy some more, 99 bottles of beer on the wall.'
+        }
+        else
+        if(i == 2)
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"